### DISCUSSION TITLE
Can this work with Redwoodjs project?

### DISCUSSION THREAD

**Discussion by sgr0691** [shadcn-ui/ui#723] [Q&A]
Hello! I was just wondering if this could work with a redwoodjs project? And if so could I get some details on how? Thank you!

**Comment by danielbolivar** [comment]
Im pretty sure they can. Think that just need to copy paste them on your project

Also you will need to install some dependencies like radix and tailwind.

You can also check the docs and do mimic how the installation works with next but on redwoodjs https://ui.shadcn.com/docs

**Reply by shansmith01** [reply]
LEaving this here incase anyone is looking https://github.com/redwoodjs/redwood/pull/10459/files
